test/js/bun/test/jest-each.test.ts

```diff
@@ -11,10 +11,10 @@ describe("jest-each", () => {
     expect(it.each).toBeTypeOf("function");
     expect(it.each([])).toBeTypeOf("function");
   });
-  it.each(NUMBERS)("add two numbers", (a, b, e) => {
+  it.each(NUMBERS)("%i + %i = %i", (a, b, e) => {
```

Comment:
There should be tests in `test/cli/test` that verifies that these labels are formatted properly, from stdout.